test/spec/algoliasearch.helper/clears.js

```diff
@@ -29,88 +28,74 @@ function fixture() {
   var helper = fixture();
 
-  t.deepEqual(helper.state.facetsRefinements, {facet1: ['0'], facet2: ['0']});
-  t.deepEqual(
-    helper.state.disjunctiveFacetsRefinements,
-    {disjunctiveFacet1: ['0'], disjunctiveFacet2: ['0']});
-  t.deepEqual(helper.state.facetsExcludes, {excluded1: ['0'], excluded2: ['0']});
-  t.deepEqual(
-    helper.state.numericRefinements,
-    {numeric1: {'>=': [0], '<': [10]}, numeric2: {'>=': [0], '<': [10]}});
-
-  t.end();
+  expect(helper.state.facetsRefinements).toEqual({facet1: ['0'], facet2: ['0']});
+  expect(helper.state.disjunctiveFacetsRefinements).toEqual({disjunctiveFacet1: ['0'], disjunctiveFacet2: ['0']});
+  expect(helper.state.facetsExcludes).toEqual({excluded1: ['0'], excluded2: ['0']});
+  expect(helper.state.numericRefinements).toEqual({numeric1: {'>=': [0], '<': [10]}, numeric2: {'>=': [0], '<': [10]}});
 });
 
-test('Clear with a name should work on every type and not remove others than targetted name', function(t) {
+test('Clear with a name should work on every type and not remove others than targetted name', function() {
   var helper = fixture();
 
   helper.clearRefinements('facet1');
-  t.deepEqual(helper.state.facetsRefinements, {facet2: ['0']});
+  expect(helper.state.facetsRefinements).toEqual({facet2: ['0']});
 
   helper.clearRefinements('disjunctiveFacet1');
-  t.deepEqual(helper.state.disjunctiveFacetsRefinements, {disjunctiveFacet2: ['0']});
+  expect(helper.state.disjunctiveFacetsRefinements).toEqual({disjunctiveFacet2: ['0']});
 
   helper.clearRefinements('excluded1');
-  t.deepEqual(helper.state.facetsExcludes, {excluded2: ['0']});
+  expect(helper.state.facetsExcludes).toEqual({excluded2: ['0']});
 
   helper.clearRefinements('numeric1');
-  t.deepEqual(helper.state.numericRefinements, {numeric2: {'>=': [0], '<': [10]}});
-
-  t.end();
+  expect(helper.state.numericRefinements).toEqual({numeric2: {'>=': [0], '<': [10]}});
 });
 
-test('Clearing the same field from multiple elements should remove it everywhere', function(t) {
+test('Clearing the same field from multiple elements should remove it everywhere', function() {
   var helper = fixture();
 
   helper.addNumericRefinement('facet1', '>=', '10').toggleExclude('facet1', 'value');
 
-  t.deepEqual(helper.state.facetsRefinements.facet1, ['0']);
-  t.deepEqual(helper.state.numericRefinements.facet1, {'>=': [10]});
-  t.deepEqual(helper.state.facetsExcludes.facet1, ['value']);
+  expect(helper.state.facetsRefinements.facet1).toEqual(['0']);
+  expect(helper.state.numericRefinements.facet1).toEqual({'>=': [10]});
+  expect(helper.state.facetsExcludes.facet1).toEqual(['value']);
 
   helper.clearRefinements('facet1');
-  t.assert(isUndefined(helper.state.facetsRefinements.facet1));
-  t.assert(isUndefined(helper.state.numericRefinements.facet1));
-  t.assert(isUndefined(helper.state.facetsExcludes.facet1));
-
-  t.end();
+  expect(isUndefined(helper.state.facetsRefinements.facet1)).toBeTruthy();
```

Comment:
The assertion can be stricter.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453